packages/grafana-ui/src/components/Logs/LogDetailsRow.tsx

```diff
@@ -17,13 +17,12 @@ import { LogLabelStats } from './LogLabelStats';
 export interface Props extends Themeable {
   parsedValue: string;
   parsedKey: string;
-  field: string;
-  row: LogRowModel;
   isLabel: boolean;
   parser?: LogsParser;
   getRows: () => LogRowModel[];
   onClickFilterLabel?: (key: string, value: string) => void;
   onClickFilterOutLabel?: (key: string, value: string) => void;
+  links?: string[];
```

Comment:
should this be [LinkModel](https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/types/dataLink.ts#L37) rather than string?  It would be nice if this supported `onClick` like the context menu links do